app/views/govuk_publishing_components/components/_accessible_autocomplete.html.erb

```diff
@@ -0,0 +1,25 @@
+<%
+  id ||= "autocomplete-#{SecureRandom.hex(4)}"
+  label ||= nil
+  data_attributes ||= nil
+  options ||= []
+  selected_option ||= nil
+%>
+
+<% if label %>
+  <%=
+    render "govuk_publishing_components/components/label", {
+      html_for: id
+    }.merge(label.symbolize_keys)
+  %>
+<% end %>
+
+<div class="gem-c-accessible-autocomplete" data-module="accessible-autocomplete">
```

Comment:
this should be:

```<%= tag.div class: "gem-c-accessible-autocomplete", "data-module": "accessible-autocomplete" do %>```

Reply:
@DilwoarH what's the benefit of doing it like this?

Reply:
Out of interest, what's the reason for using `tag.div`? I referenced a few of the existing components when developing this new one, which used a similar `<div>` declaration to the above.